tests/components/vlc_telnet/test_config_flow.py

```diff
@@ -299,22 +302,24 @@ async def test_hassio_flow(hass: HomeAssistant) -> None:
 
         assert result2["type"] == RESULT_TYPE_CREATE_ENTRY
         assert result2["title"] == test_data["name"]
-        assert result2["data"] == test_data
+        assert result2["data"] == test_data.config
         assert len(mock_setup_entry.mock_calls) == 1
 
 
 async def test_hassio_already_configured(hass: HomeAssistant) -> None:
     """Test successful hassio flow."""
 
-    entry_data = {
-        "password": "test-password",
-        "host": "1.1.1.1",
-        "port": 8888,
-        "name": "custom name",
-        "addon": "vlc",
-    }
+    entry_data = HassioServiceInfo(
```

Comment:
This is not config entry data. This is discovery info.
```suggestion
    discovery_info = HassioServiceInfo(
```

Reply:
You are right - I have corrected it